MUXSDKStats/Podfile

```diff
@@ -1,12 +1,13 @@
 # Uncomment the next line to define a global platform for your project
-# platform :ios, '9.0'
+platform :ios, '9.0'
```

Comment:
Is this going to force us to iOS 9+? And if so, is that what the IMA SDK requires?

Reply:
the IMA SDK has a minimum iOS9+ requirement. that's why the uncomment

Reply:
ahh interesting, okay